### ISSUE TITLE
Dark mode

### ISSUE THREAD

**Issue by pvinis** [react-native-community/upgrade-helper#97]
# Feature Request

**Comment by lucasbento** [comment]
I'm spending some time on this, what I got so far:

![Upgrade-Helper with Dark Mode](https://user-images.githubusercontent.com/6207220/65038938-60231100-d951-11e9-9b7b-697017ed9329.png)

![Upgrade-Helper with Dark Mode and the from dropdown version opened](https://user-images.githubusercontent.com/6207220/65038950-6913e280-d951-11e9-8ace-f0bda9e7c09f.png)

I would really appreciate feedback on the colors, I'm basically just trying stuff until it looks good, still have quite some stuff to do though.

/cc @pvinis @kelset @KevinHu2014 @vinnyA3 @iamrestrepo

**Comment by lucasbento** [comment]
As a reference, this is how a GitHub PR looks like when using [GitHub-Dark](https://github.com/StylishThemes/GitHub-Dark):

![PR on GitHub with GitHub-Dark](https://user-images.githubusercontent.com/6207220/65039452-9ad97900-d952-11e9-9cdc-36467a4e7619.png)

**Comment by Johan-dutoit** [comment]
I'd only suggest to improve accessibility ![Screenshot 2019-09-17 at 13 13 47](https://user-images.githubusercontent.com/1611055/65040753-0587b600-d94d-11e9-8d88-24d690113253.png)

**Comment by lucasbento** [comment]
@Johan-dutoit: thanks for the tip!

What tool are you using to check that?

**Comment by Johan-dutoit** [comment]
I just used https://webaim.org/resources/contrastchecker/

**Comment by vinnyA3** [comment]
@Johan-dutoit useful tool! thank you for sharing 😄

**Comment by vinnyA3** [comment]
@lucasbento Looks awesome!  My eyes thank you lol.
Kind of sort of tangential, but where are you planning on putting the toggle?  I was thinking perhaps it would be nice to put it in the settings dropdown.

**Comment by lucasbento** [comment]
@vinnyA3: yeah, I want to put on the settings but still need to look into it.

**Comment by KevinHu2014** [comment]
@lucasbento Wow!  It looks Awesome 👍

**Comment by SaeedZhiany** [comment]
is there any progress on this?

**Comment by pvinis** [comment]
not yet. I might spend some time soon. Until then, I recommend https://darkreader.org/.
